python/ray/autoscaler/_private/load_metrics.py

```diff
@@ -97,10 +97,12 @@ def update(
         infeasible_bundles: List[Dict[str, float]] = None,
         pending_placement_groups: List[PlacementGroupTableData] = None,
         cluster_full_of_actors_detected: bool = False,
+        node_last_used_time_s: Optional[float] = None,
     ):
         self.static_resources_by_ip[ip] = static_resources
         self.raylet_id_by_ip[ip] = raylet_id
         self.cluster_full_of_actors_detected = cluster_full_of_actors_detected
+        self.ray_nodes_last_used_time_by_ip[ip] = node_last_used_time_s
```

Comment:
In which cases will `node_last_used_time_s` not be set? If `node_last_used_time_s` is `None`, an error may occur when performing `time (float) - None`.

Reply:
Check added on https://github.com/ray-project/ray/pull/48519/commits/be86afdfb5ad3554a2ba4ad4298310458e4b8871